[2022-02-02 21:28:50,078] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-02 21:28:50,128] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-02 21:28:50,129] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:28:50,129] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-02 21:28:50,130] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:28:50,170] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-12-02 06:00:00+00:00
[2022-02-02 21:28:50,178] {standard_task_runner.py:52} INFO - Started process 1494 to run task
[2022-02-02 21:28:50,183] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-12-02T06:00:00+00:00', '--job-id', '358', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmpa3lk_epr', '--error-file', '/tmp/tmpnt6f70ej']
[2022-02-02 21:28:50,185] {standard_task_runner.py:77} INFO - Job 358: Subtask local_to_gcs_task
[2022-02-02 21:28:50,380] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [running]> on host d574c7c5b085
[2022-02-02 21:28:50,551] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-02 21:28:50,607] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-12-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-12-02T06:00:00+00:00
[2022-02-02 21:30:06,677] {python.py:175} INFO - Done. Returned value was: None
[2022-02-02 21:30:06,722] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20191202T060000, start_date=20220202T212850, end_date=20220202T213006
[2022-02-02 21:30:06,833] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 21:30:06,959] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 11:47:07,740] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-09 11:47:07,785] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-09 11:47:07,785] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 11:47:07,786] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-09 11:47:07,786] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 11:47:07,828] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-12-02 06:00:00+00:00
[2022-02-09 11:47:07,835] {standard_task_runner.py:52} INFO - Started process 1310 to run task
[2022-02-09 11:47:07,846] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-12-02T06:00:00+00:00', '--job-id', '430', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmpx9rprwxq', '--error-file', '/tmp/tmp24ik76ah']
[2022-02-09 11:47:07,847] {standard_task_runner.py:77} INFO - Job 430: Subtask local_to_gcs_task
[2022-02-09 11:47:08,003] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [running]> on host c5795c3f6004
[2022-02-09 11:47:08,148] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 11:47:08,211] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-12-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-12-02T06:00:00+00:00
[2022-02-09 11:47:56,081] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 11:47:56,121] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20191202T060000, start_date=20220209T114707, end_date=20220209T114756
[2022-02-09 11:47:56,221] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 11:47:56,312] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 01:15:55,261] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-10 01:15:55,310] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-10 01:15:55,310] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 01:15:55,310] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 01:15:55,311] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 01:15:55,375] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-12-02 06:00:00+00:00
[2022-02-10 01:15:55,385] {standard_task_runner.py:52} INFO - Started process 7591 to run task
[2022-02-10 01:15:55,392] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-12-02T06:00:00+00:00', '--job-id', '498', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmp3jfz2rjb', '--error-file', '/tmp/tmp4x86ovkx']
[2022-02-10 01:15:55,394] {standard_task_runner.py:77} INFO - Job 498: Subtask local_to_gcs_task
[2022-02-10 01:15:55,554] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [running]> on host 0fd4ea805f5e
[2022-02-10 01:15:55,684] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 01:15:55,736] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-12-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-12-02T06:00:00+00:00
[2022-02-10 01:17:08,673] {python.py:175} INFO - Done. Returned value was: None
[2022-02-10 01:17:08,708] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20191202T060000, start_date=20220210T011555, end_date=20220210T011708
[2022-02-10 01:17:08,783] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 01:17:08,877] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-19 01:55:46,752] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-19 01:55:46,806] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-19 01:55:46,807] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 01:55:46,807] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 01:55:46,807] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 01:55:46,848] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-12-02 06:00:00+00:00
[2022-02-19 01:55:46,855] {standard_task_runner.py:52} INFO - Started process 4392 to run task
[2022-02-19 01:55:46,862] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-12-02T06:00:00+00:00', '--job-id', '659', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmpqcmpfl65', '--error-file', '/tmp/tmps17x3rd9']
[2022-02-19 01:55:46,864] {standard_task_runner.py:77} INFO - Job 659: Subtask local_to_gcs_task
[2022-02-19 01:55:47,027] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [running]> on host 312f1fe573aa
[2022-02-19 01:55:47,162] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-19 01:55:47,258] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-12-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-12-02T06:00:00+00:00
[2022-02-19 01:57:00,995] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 01:57:01,040] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20191202T060000, start_date=20220219T015546, end_date=20220219T015701
[2022-02-19 01:57:01,133] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 01:57:01,243] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-19 11:55:23,139] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-19 11:55:23,209] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [queued]>
[2022-02-19 11:55:23,209] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 11:55:23,209] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 11:55:23,210] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 11:55:23,275] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-12-02 06:00:00+00:00
[2022-02-19 11:55:23,284] {standard_task_runner.py:52} INFO - Started process 5235 to run task
[2022-02-19 11:55:23,301] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-12-02T06:00:00+00:00', '--job-id', '1029', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmp8z62c056', '--error-file', '/tmp/tmprxpei3ps']
[2022-02-19 11:55:23,303] {standard_task_runner.py:77} INFO - Job 1029: Subtask local_to_gcs_task
[2022-02-19 11:55:23,502] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-12-02T06:00:00+00:00 [running]> on host f81f14254480
[2022-02-19 11:55:23,613] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-19 11:55:23,669] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-12-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-12-02T06:00:00+00:00
[2022-02-19 11:56:44,280] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 11:56:44,315] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20191202T060000, start_date=20220219T115523, end_date=20220219T115644
[2022-02-19 11:56:44,406] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 11:56:44,539] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
